docs/developer_manual/installation.rst

```diff
@@ -43,3 +55,12 @@ Add the current directory to the Python path with the following command::
 (This last two commands are included into the ``Makefile`` of the main directory.
 They can be run by ``make develop``.
 The command ``make clean`` deletes the binaries.)
+
+You will also likely want to install Capytaine's optional dependencies::
+
+	conda install matplotlib vtk pytest quadpy
+	pip install pygmsh 
```

Comment:
Also `ipython` and `hypothesis` to run all the tests?

Reply:
right! resolved w. 109f63dc5949a95e188f5c9afa78e5c4feb512d2